src/include/cell_inline.h

```diff
@@ -970,6 +970,11 @@ __cell_unpack_window_cleanup(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk
      * No delete              txnid=MAX, ts=MAX,            txnid=MAX, ts=MAX,
      *                        durable_ts=NONE               durable_ts=NONE
      */
+    /* Don't reset the transaction ids in rollback to stable when called from recovery. */
```

Comment:
Can you add more details here like why we shouldn't reset the transactions for more clarity?